tests/Properties.hs

```diff
@@ -172,6 +172,16 @@ isObjectWithSingleField :: Value -> Bool
 isObjectWithSingleField (Object obj) = H.size obj == 1
 isObjectWithSingleField _            = False
 
+-- | EitherTextInt: String, Number or Array
+isUntaggedValue :: Value -> Bool
+isUntaggedValue (String s)
+    | s == "nonenullary"   = True
+isUntaggedValue (Bool _)   = True
+isUntaggedValue (Number _) = True
+isUntaggedValue (Array a)  = length a == 2
+isUntaggedValue _          = False
```

Comment:
Ah, OK. I was confused since the implementation doesn't quite match the documentation (it says `String, Number or Array`, but it also checks for `Bool`).